index.html

```diff
@@ -8,7 +8,29 @@
     <title>Microverse</title>
 </head>
 <body>
-    <h1>Hello Microverse</h1>
+    <div class="nav-bar">
+        <div class="logo">Yash Solo</div>
+        <div class="ham-manu"><img src="res/ham-manu.svg" alt="ham-menu"></div>
+    </div>
+    <div class="hero-sec">
+        <div class="hero-main-header"><h1>Hey There.</br>
+            I’m Yash</h1></div>
+        <div class="hero-sm-header"><h3>I’am a Software Developer</h3></div>
+        <div class="hero-text"><p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p></div>
+        <div class="social-media-sec">
+            <img class="social-ico" src="res/git.svg" alt="git">
+            <img class="social-ico" src="res/linkedin.svg" alt="linkedin">
+            <img class="social-ico" src="res/vic.svg" alt="victory">
+            <img class="social-ico" src="res/twitter.svg" alt="twitter">
+            <img class="social-ico" src="res/m.svg" alt="meta">
+        </div>
+        
+        <div class="illustration">
+            <img src="res/Illustration.svg" alt="illustration">
+        </div>
```

Comment:
- [ ]  Kyndly use css background image for decorations. Decorations are always background images, not `img` tags.

- If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.